src/edgeDebugSession.ts

```diff
@@ -2,14 +2,14 @@
  * Copyright (C) Microsoft Corporation. All rights reserved.
  *--------------------------------------------------------*/
 
-import {ChromeDebugSession} from 'vscode-chrome-debug-core';
+import {ChromeDebugSession, IChromeDebugSessionOpts} from 'vscode-chrome-debug-core';
 import {EdgeDebugAdapter} from './edgeDebugAdapter';
 
 export class EdgeDebugSession extends ChromeDebugSession {
 
     public constructor(targetLinesStartAt1: boolean, isServer: boolean = false) {
```

Comment:
Is there a particular harm in doing it this way? The reason I ask is I am using the class in edgeDebugAdapter.ts as well in the constructor. I can do the reformatting if there is a good reason though.